uniffi_bindgen/src/interface/error.rs

```diff
@@ -135,6 +135,20 @@ impl Error {
     }
 }
 
+impl From<uniffi_meta::ErrorMetadata> for Error {
+    fn from(meta: uniffi_meta::ErrorMetadata) -> Self {
+        let flat = meta.variants.iter().all(|v| v.fields.is_empty());
```

Comment:
I found this is actually wrong, and am working on a fix (I'm doing a separate commit though, so review-ability should not be harmed).